enterprise/kernel/src/test/java/org/neo4j/util/JvmRunner.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.util;
+
+import org.junit.Rule;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.neo4j.commandline.admin.AdminTool;
+import org.neo4j.io.proc.ProcessUtil;
+import org.neo4j.test.ProcessStreamHandler;
+import org.neo4j.test.rule.TestDirectory;
+
+public class JvmRunner
```

Comment:
Seems pretty backup specific. Maybe just add them as more static methods to `org.neo4j.backup.util.TestHelpers`?

Reply:
Because it contains `@Rule TestDirectory` then it is expected to be extended from. The method that doesn't require `NEO4J_HOME` depends on the TestDirectory. Also it had to be moved to enterprise/kernel because there are tests in causal clustering (if I recall) that use this as well. Its a maven tree thing essentially

Reply:
`OnlineBackupCommandCcIT` is the only test that uses that feature of it, and it could just as well have its own `TestDirectory`.